### PR TITLE
V13 RC3: login screen design update

### PR DESCRIPTION
**Pull request by iOvergaard**
## Description

Update articles related to: login, external login, and two-factor in preparation for V13 RC3.

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [x] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
13.0.0

## Deadline (if relevant)

ASAP for RC3

### CODE REVIEW

File: .github/styles/UmbracoDocs/Acronyms.yml [hunk 1]
```diff
@@ -80,3 +80,5 @@ exceptions:
   - GIF
   - GDPR
   - NPM
+  - SMTP
```
Comment (jonat123) [inline .github/styles/UmbracoDocs/Acronyms.yml:83]:
```suggestion

```

**Review by jonat123 — CHANGES_REQUESTED** [review]
@iOvergaard Looks good overall however I noticed that you added SMTP to the acronyms list. I've added a suggestion to remove it as it is something that maybe not everyone knows what it stands for when they are browsing our documentation, so we should definitely still be defining it  🙂

**Comment by iOvergaard** [comment]
> @iOvergaard Looks good overall however I noticed that you added SMTP to the acronyms list. I've added a suggestion to remove it as it is something that maybe not everyone knows what it stands for when they are browsing our documentation, so we should definitely still be defining it 🙂

Hi, the problem with defining it is that it stands for "Simple Mail Transfer Protocol" and Vale starts complaining about the word "simple" in this context. Please advise how to circumvent this ☺️

File: 13/umbraco-cms/fundamentals/backoffice/login.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ The **Forgotten password?** link allows your backoffice users to reset their pas
 
 Set it to `true` to enable the password reset feature, and `false` to disable the feature.
 
-You will also need to configure an Simple Mail Transfer Protocol (SMTP) server in your `appsettings.json` file. When you get a successful result on the SMTP configuration when running a health check in the backoffice, you are good to go!
+You will also need to configure an outbound e-mail server (SMTP) in your `appsettings.json` file. When you get a successful result on the SMTP configuration when running a health check in the backoffice, you are good to go!
```
Comment (iOvergaard) [inline 13/umbraco-cms/fundamentals/backoffice/login.md:59]: @jonat123 Before it used both the long form of the technology "Simple Mail Transfer Protocol" and the acronym "SMTP", but that didn't work with Vale, nor does this work now without being allowed the "SMTP" acronym. Can you give me some tips to describe this in a better way, please?

**Comment by jonat123** [comment]
@iOvergaard Since it is a part of explaining an acronym, then it is okay to just ignore the warning 😊

File: 13/umbraco-cms/fundamentals/backoffice/login.md (hunk 2 above)
Comment (jonat123) [reply 13/umbraco-cms/fundamentals/backoffice/login.md:59]: In this case I think you can just use the initial version with the Simple Mail Transfer Protocol (SMTP) definition.

As simple is a part of the acronym the warning in this case should just be ignored 😊

Comment (iOvergaard) [reply 13/umbraco-cms/fundamentals/backoffice/login.md]: Fixed, thanks!

**Review by jonat123 — APPROVED** [review]
@iOvergaard Awesome! 
We will make sure this is merged in when the RC 3 is released 😄

**Comment by iOvergaard** [comment]
@jonat123 RC3 was released yesterday. Please merge when you get a chance :)

Also I reverted a few commits that belong to RC4, so it should be good to go now!

**Comment by jonat123** [comment]
@iOvergaard Great, I will merge this in right away!

**Pull request merged**